src/ledger/test/LedgerManagerTests.cpp

```diff
@@ -63,45 +62,4 @@ class LedgerManagerTestApplication : public TestApplication
         return std::make_unique<LedgerManagerForTests>(*this);
     }
 };
-}
-
-TEST_CASE("new ledger comes from network after last applyBufferedLedgers is "
```

Comment:
@marta-lokhova I removed this test case because LedgerManager no longer applies the buffered ledgers. I think the "catchup with a gap" test in HistoryTests covers this case now, but I'm not 100% sure.